migrations/20161021002706-create-topic.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+
+module.exports = {
+  up: function(queryInterface, Sequelize) {
+
+    const createUserTopicTable = () => {
+      return queryInterface.createTable('UserTopic', {
+        id: {
+          allowNull: false,
+          autoIncrement: true,
+          primaryKey: true,
+          type: Sequelize.INTEGER
+        },
+        UserId: {
```

Comment:
I would like to use a consistent naming scheme in the database - lower snake case: `user_id`, `topic_id`, etc.  An exception is the auto generated fields like `createdAt` and `updatedAt` (though bonus points if you could make these consistent with our naming convention as well).